### DISCUSSION TITLE
Title feature

### DISCUSSION THREAD

**Discussion by chlanz** [olvidalo/obsidian-chord-sheets#3] [Ideas]
Hi and thanks for this plugin! Could you maybe implement some sort of title function so that I could name a section of a song (for instance "chorus") and later use just the title again without copying everything again and again.

**Comment by olvidalo** [comment]
Hmm, do you mean something like re-usable snippets?

**Reply by chlanz** [reply]
Hi, I'm thinking of a feature similar to this:
![Bildschirmfoto 2024-08-22 um 15 40 55](https://github.com/user-attachments/assets/1a2decc4-7128-43c8-927d-a2088b32d05e)
BTW I like everything else within your plugin much more :)

**Comment by olvidalo** [comment]
Highlighting of section headers is supported in the latest release! If you need custom control over colors, font weight and font size for headers, you can use [Style Settings](https://github.com/mgmeyers/obsidian-style-settings).

![image](https://github.com/user-attachments/assets/2b1446e9-70d6-43e0-a414-a9752c385957)

**Reply by chlanz** [reply]
Hi and thank you! I must be doing something wrong because the following lines:

```chords
[Intro]
Em Em7 D Em (2x)
Em C D G (x3)
[1. Verse]
Em        C     D         G 
I want to live, I want to give
```

display like this:
![image](https://github.com/user-attachments/assets/f1322051-118c-444d-ac55-f28f80dcba22)
using version 1.2.0

**Reply by olvidalo** [reply]
You mean the header color? It depends on the theme which color is used. Some themes have a "bold color", such as Minimal which is used in the screenshot above. Headers are highlighted in this color. If there is no bold color, it uses the theme's accent color that is also used for the chords.

Or am I missing something? How would you expect your snipped to be displayed?

**Reply by chlanz** [reply]
I see, I was not expecting to see the brackets but rather tat the text between the brackets would be bold, colored or highlighted... I understand I can set "bold color" to get that but the brackets will stay - right?

**Reply by olvidalo** [reply]
Ah, ok – yep, the brackets will stay. Per default their color is toned down a bit to make the text between them stand out more. But they are not hidden because monospaced code blocks are supposed to show everything as-is, unlike how Obsidian handles it outside code blocks.

**Reply by chlanz** [reply]
It works fabulous! Thank you!
